### ISSUE TITLE
Give every resource an "ore" tile.

### ISSUE THREAD

**Issue by SlitheryC** [Anuken/Mindustry-Suggestions#2777]
### Describe the content or mechanics you are proposing.

Manufacturable resources have their own drillable "ores" in the editor.

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

I feel like we need more options in the editor to motivate players to actively push constantly instead of just building big defenses and spamming units in Attack PvP. while it isnt a bad thing, and is basically the whole point, we need something to fight for aside from just victory in some maps.

with the current usable tiles in editor, it can be pretty difficult to make an area valuable unless you outright restrict important resources like thorium, without making it take up a quarter of the map.

This is why I think we should add manufactured resource "ores-" they dont outright limit you from simply making a great silicon or plastanium making system, but theyll certainly make it easier to get those resources and certainly save room, which can be vital in some maps, especially those in space. 

it would also, in general, help diversify or restrict options- coal can be used for quite a few things, energy generation, silicon, graphite, pyrite, etc, but a pvp map where graphite or silicon, or maybe even blast compound or pyrite is common but theres almost no coal around would be very interesting and a give a unique experience.

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by joshuaptfan** [comment]
Uh yeah, sure, let's make factories irrelevant. Why do we even need those? And plastanium is a naturally occurring substance, right?

**Comment by MHeMoTexHuK** [comment]
How would absolutely synthetic stuff (and especially alloys) like surge, phase, metaglass or plastanium suddenly appear in the ground? It's not like we can simply mine steel or extract plastic on earth

**Comment by IIINSTINCT** [comment]
Mods

**Comment by OnyxFinger** [comment]
I agree on "mods" this time, also if many players are turtling either you're letting them do so which you can avoid, or you're playing on servers such as io PvP and hardcore PvP which promote it cuz funee and fuck good gameplay that doesn't take ages.
